src/sentry/utils/eventuser.py

```diff
@@ -24,7 +35,8 @@ class EventUser:
     username: str
     name: str
     ip_address: str
-    id: Optional[int] = None
+    user_id: int
```

Comment:
Does making this required break any existing code? Any chance we're passing something that was intended as `id` and it's now going to `user_id`?

Reply:
My point is that previously, you might have been passing `EventUser(1, "my@email.com", "user", "Dan Fuller", "192.168.0.1", 123)`. You've now added `user_id` to the definition before `id`, so in my previous example, `123` would now be passed to `user_id` instead of `id`. I just want to make sure that's not having any unintended side effects.

Reply:
No need to switch them around if you've checked the usage, just wanted to be sure